dist/images/start-ovs-dpdk.sh

```diff
@@ -12,6 +12,7 @@ CONFIG_FILE=/opt/ovs-config/config.cfg
 
 # Check if config file exists, create default one if not
 if ! test -f "$CONFIG_FILE"; then
+	mkdir -p $(dirname ${CONFIG_FILE})
```

Comment:
I hit a small bug while testing the new smaller dpdk image. The start-ovs-dpdk.sh script in my new image contained the changes from 8121afd6037a3414b2ca1a0ff8f3f004d47695a7 but the install.sh script from release 1.2 did not contain the "mountPath" that is responsible for creating the /opt/ovs-config/ directory within the ovs-ovn pod. The print command errored due to non existing directory and ovs-ovn pod failed to start.

@oilbeater This bug is fixed, but it made me think, since the dpdk images are meant to work independently of any particular kube-ovn version, I wonder should we update the dpdk.md so the user is instructed to download install.sh from master rather than any version?
`wget https://raw.githubusercontent.com/alauda/kube-ovn/master/dist/images/install.sh`

Bug is fixed and I'm happy to leave the doc as is, but just a thought.

Reply:
Other kube-ovn components images in master are not stable, and may lead other issues. I'd prefer to add another variable in `install.sh` for dpdk image, and mention the latest image tag in `dpdk.md`

Reply:
I agree.